docpages/example_programs/misc/using-emojis.md

```diff
@@ -0,0 +1,29 @@
+\page using-emojis Using Emojis
+
+Need your bot to use an emoji? Then you've come to the right place! Here are three examples of using emojis.
+
+\note If your bot isn't in a guild where you want to take the custom emoji from, it won't work. If you're trying to send one, it'll turn into `:name:`, otherwise it won't work at all giving you the Unknown Emoji error.
```

Comment:
The wording here is a bit odd. Correct to:
```
\note If your bot isn't in the guild where you want to use the custom emoji from, it won't work, giving you dpp::exception_error_code::err_unknown_emoji.
```

Reply:
Yeah I'll just yoink that

Reply:
That was the point lol